jobtracker.go

```diff
@@ -90,18 +90,20 @@ type jobTracker struct {
 	rm       string
 	hs       string
 	ps       string
+	namenodeAddress string
```

Comment:
i left the vertical alignment of the surrounding lines alone to keep the diff small, and b/c the future of the namenode dependency being undecided.